src/os/alert/alerteventlevel.js

```diff
@@ -0,0 +1,36 @@
+goog.module('os.alert.AlertEventLevel');
+goog.module.declareLegacyNamespace();
```

Comment:
Is calling this function in every class needed? What does it actually do? It doesn't have any arguments... so how does it know the context of the declaration we're making?

Reply:
Yep. This is what actually allows you to `goog.require` the module in `goog.provide` files. `goog.module` doesn't add anything in the global namespace (part of the point of modules), so `goog.module.declareLegacyNamespace` does that if called within a module.

You can dig into the details in Closure's code if you want, but basically they have a module loader that handles each of these files and calling this function during module load puts the namespace on the global window object.